src/main/resources/pdiresources/jobs/control-center.kjb

```diff
@@ -316,7 +316,7 @@
       <attributes/>
       <specification_method>filename</specification_method>
       <job_object_id/>
-      <filename>pdiresources/jobs/migration.kjb</filename>
+      <filename>${Internal.Entry.Current.Directory}/migration.kjb</filename>
```

Comment:
@kaweesi ${Internal.Entry.Current.Directory} seems to only work with spoon. This causes null pointer exceptions in the tool.